.github/workflows/pytest.yml

```diff
@@ -32,6 +32,8 @@ jobs:
             [ -f "$HOME/.ocean/ocean-contracts/artifacts/ready" ] && break
             done
           ls -la "$HOME/.ocean/ocean-contracts/artifacts/"
+        env:
```

Comment:
should this stay or should subgraph be updated?

Reply:
finally, barge was updated, so I have removed this